nl_server/routes.py

```diff
@@ -35,7 +35,8 @@
 
 @bp.route('/healthz')
 def healthz():
-  default_index_type = current_app.config[config.DEFAULT_INDEX_TYPE_KEY]
+  default_index_type = current_app.config[
+      config.EMBEDDINGS_SPEC_KEY]['default_index']
```

Comment:
Don't crash (bug from yesterday).  How about, if 'default_index' is empty, return Service Unavailable?

Reply:
done